checks/remotesettings/changes_timestamps.py

```diff
@@ -30,10 +31,21 @@ def get_timestamp(client, bucket, collection, timestamp):
         for entry in entries
     ]
     results = await asyncio.gather(*futures)
-    failing = []
+    all_good = True
+    datetimes = []
     for (entry, timestamp) in zip(entries, results):
+        collection_timestamp = timestamp
+        entry_timestamp = str(entry["last_modified"])
+        all_good = all_good and collection_timestamp == entry_timestamp
```

Comment:
This is a little fancy. I'd rather see `all_good` be calculated declaratively outside the for loop.